docs/tutorial/part-one/index.md

```diff
@@ -20,7 +20,7 @@ When creating a new Gatsby site, you can use the following command structure to
 gatsby new [SITE_DIRECTORY_NAME] [URL_OF_STARTER_GITHUB_REPO]
 ```
 
-If you omit a URL from the end, Gatsby will automatically generate a site for you based on the [**default starter**](https://github.com/gatsbyjs/gatsby-starter-default). For this section of the tutorial, stick with the “Hello World” site you already created in tutorial part zero.
+If you omit a URL from the end, Gatsby will automatically generate a site for you based on the [**default starter**](https://github.com/gatsbyjs/gatsby-starter-default). For this section of the tutorial, stick with the “Hello World” site you already created in tutorial part zero. You can learn more about [modifying starters](/docs/modifying-a-starter) in the docs.
```

Comment:
We've talked at times about limiting links in the tutorial to keep learners focused. This one is really compelling to add, however, as it's relevant to the task at hand (and people asked us for it in a Getting Started unconference session). My instinct is to keep it in, as there isn't a very good spot for it at the end of Tutorial Part One. It seems like it fits the best here.

Reply:
Thanks! I did see a note about avoiding linking away from the tutorial. If you hadn’t mentioned it as an option in the related issue, I probably would’ve left this out. Especially given that new-to-Gatsby folks seem to be wondering  _how_ to modify a starter (I asked around a bit before writing this), I thought it would be useful here.

Reply:
Sounds wise! Let's keep it.